javers-core/src/main/java/org/javers/core/json/typeadapter/commit/CdoSnapshotTypeAdapter.java

```diff
@@ -54,7 +55,12 @@ public CdoSnapshot fromJson(JsonElement json, JsonDeserializationContext context
         CdoSnapshotBuilder cdoSnapshotBuilder = cdoSnapshot(cdoId, commitMetadata);
         cdoSnapshotBuilder.withCommitMetadata(commitMetadata);
 
-        JsonObject state = jsonObject.get(STATE).getAsJsonObject();
+        JsonElement initial = jsonObject.get(INITIAL_NAME);
```

Comment:
nie powinno, jesli pola nie będzie w jsonie to w CdoSnapshot będzie defaultowa wartość czyli false, czyli ok. See CdoSnapshotTypeAdapter line 59